packages/account/src/Components/currency-selector/currency-selector.tsx

```diff
@@ -191,29 +223,28 @@ const CurrencySelector = ({
                                     {!!reorderCurrencies(crypto, 'crypto')?.length && (
                                         <React.Fragment>
                                             <RadioButtonGroup
-                                                id='currency'
                                                 className='currency-selector__radio-group currency-selector__radio-group--with-margin'
                                                 label={localize('Cryptocurrencies')}
-                                                value={values.currency}
-                                                error={errors.currency}
-                                                touched={touched.currency}
                                                 item_count={reorderCurrencies(crypto, 'crypto').length}
                                                 description={description}
                                             >
-                                                {reorderCurrencies(crypto, 'crypto').map(currency => (
-                                                    <Field
-                                                        key={currency.value}
-                                                        component={RadioButton}
-                                                        selected={
-                                                            available_crypto_currencies?.filter(
-                                                                ({ value }) => value === currency.value
-                                                            )?.length === 0
-                                                        }
-                                                        name='currency'
-                                                        id={currency.value}
-                                                        label={currency.name}
-                                                    />
-                                                ))}
+                                                {reorderCurrencies(crypto, 'crypto').map(
+                                                    (currency: TCurrencyConfig) => (
+                                                        <Field
```

Comment:
I imagine if we have types for this and other similar in this file functions we shouldn’t set types for function inside map or filter. 
should we do it now here? :)

Reply:
Yes. The obj structure is same as the mentioned type hence it is provided as type inside `map()`